client/connector.go

```diff
@@ -90,6 +98,13 @@ type connector struct {
 	enumDefaultValueMappings map[string]map[int]int
 }
 
+// connector is the ares connector implementation
+type connector struct {
+	*UpsertBatchBuilderImpl
```

Comment:
UpsertBatchBuilder interface?

Reply:
add interface UpsertBatchBuilder to separate connector from upsertBatchBuild preparation